File: hashtag-generator/hashtagGenerator.js
const generateHashtag = (string) => {

    if(!/[\sA-Za-z]/g.test(string.charAt(0))) return false;
    
    let split = string.split(" ");
    let output = [];
    let countSpaces = 0;

    for(let i = 0; i < string.length; i++) {
        if(string[i] === " ") countSpaces++;
    };
    if(countSpaces === string.length) return false;
    
    split.forEach((element) => {
        output.push(element.charAt(0).toUpperCase() + element.slice(1, element.length));
    });
    output.unshift("#");

    if(output.join("").length > 140) return false;

    return output.join("");
};

module.exports = { generateHashtag };

//https://www.codewars.com/kata/52449b062fb80683ec000024/train/javascript

File: python-venv/hashtag_generator/hashtag_generator.py
def hashtag_generator(string):

    if len(string) < 1:
        return False
    
    split = string.split(" ")
    output = []

    for i in range(len(split)):
        if len(split[i]) != 0:
            output.append(split[i][0].upper() + split[i][1:].lower())

    if len("#" + "".join(output)) > 140:
        return False
    else:
       return "#" + "".join(output)


#https://www.codewars.com/kata/52449b062fb80683ec000024/train/python